### PR TITLE
Config sample Bazel

### PR DESCRIPTION
**Pull request by cdsap**
Common setup of Bazel builds that are using Develocity

### CODE REVIEW

File: common-gradle-enterprise-bazel-configuration/.bazelrc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+common:develocity --noslim_profile
+
+build --config=develocity
+query --config=develocity
+
+# If your Develocity server forbids anonymous cache writes or scan publishing, the following 
+# additional steps are required:
+# 1. Log into your Develocity server, from the upper-right corner click your user icon, then "My Settings".
+# 2. Ensure that your user has at least "Publish build scans" and either "Read and write build cache data" or 
+#    Read build cache data and write Bazel CAS data" permissions.
```
Comment (erichaagdev) [inline common-gradle-enterprise-bazel-configuration/.bazelrc:16]: Missing opening quotation mark.

```suggestion
#    "Read build cache data and write Bazel CAS data" permissions.
```

Comment (cdsap) [reply common-gradle-enterprise-bazel-configuration/.bazelrc]: thanks @erichaagdev, good catch

File: common-gradle-enterprise-bazel-configuration/.bazelrc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+common:develocity --remote_cache=grpcs://<<develocity-server>> # adjust to your Develocity server
+common:develocity --bes_results_url=https://<<develocity-server>>/build/ # adjust to your Develocity server
+common:develocity --bes_backend=grpcs://<<develocity-server>> # adjust to your Develocity server
+common:develocity --build_event_publish_all_actions=true
+common:develocity --noslim_profile
+
+build --config=develocity
+query --config=develocity
+
+# If your Develocity server forbids anonymous cache writes or scan publishing, the following
```
Comment (runningcode) [inline common-gradle-enterprise-bazel-configuration/.bazelrc:12]: Let's assume they are forbidding anonymous cache writes.

File: common-gradle-enterprise-bazel-configuration/.bazelrc [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# To publish Build Scans to your Develocity server, add the following to the .bazelrc file in your 
+# build workspace:
+common:develocity --remote_cache=grpcs://<<develocity-server>> # adjust to your Develocity server
+common:develocity --bes_results_url=https://<<develocity-server>>/build/ # adjust to your Develocity server
+common:develocity --bes_backend=grpcs://<<develocity-server>> # adjust to your Develocity server
+common:develocity --build_event_publish_all_actions=true
+common:develocity --noslim_profile
+
+build --config=develocity
+query --config=develocity
```
Comment (runningcode) [inline common-gradle-enterprise-bazel-configuration/.bazelrc:10]: Where does the project name get set?

File: common-gradle-enterprise-bazel-configuration/.bazelrc (hunk 2 above)
Comment (cdsap) [reply common-gradle-enterprise-bazel-configuration/.bazelrc]: done, new text added

File: common-gradle-enterprise-bazel-configuration/.bazelrc (hunk 3 above)
Comment (cdsap) [reply common-gradle-enterprise-bazel-configuration/.bazelrc:12]: issue has been fixed

Comment (runningcode) [reply common-gradle-enterprise-bazel-configuration/.bazelrc:12]: does that set the project name or the project id?

Comment (cdsap) [reply common-gradle-enterprise-bazel-configuration/.bazelrc:12]: following [Setting the project name](https://docs.gradle.com/enterprise/bazel-configuration/#project-name) it sets the project name, but in the next section https://docs.gradle.com/enterprise/bazel-configuration/#project-level-access-control it states the same properties are used for the project id

Comment (runningcode) [reply common-gradle-enterprise-bazel-configuration/.bazelrc:12]: Can you remove it for now? Following an internal discussion, it looks like there is no way to set the project name.

Comment (cdsap) [reply common-gradle-enterprise-bazel-configuration/.bazelrc:10]: removed

**Review by runningcode — APPROVED** [review]
Looks good to me, can you please check with the Bazel team to review this?

File: common-gradle-enterprise-bazel-configuration/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+## Common Gradle Enterprise Bazel configuration
```
Comment (dotta) [inline common-gradle-enterprise-bazel-configuration/README.md:1]: "Gradle Enterprise" -> "Develocity" everywhere in this file

**Review by dotta — APPROVED** [review]
LGTM

File: common-gradle-enterprise-bazel-configuration/README.md (hunk 4 above)
Comment (runningcode) [reply common-gradle-enterprise-bazel-configuration/README.md:1]: This is a good point. We will do this wholistically across the entire repo after the next release. For now let's keep it consistent.

File: common-gradle-enterprise-bazel-configuration/.bazelrc [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+# Because Bazel build scans are enriched by allowing users to upload data to the remote cache, the following
+# additional steps are required:
+# 1. Log into your Develocity server, from the upper-right corner click your user icon, then "My Settings".
+# 2. Ensure that your user has at least "Publish build scans" and either "Read and write build cache data" or 
+#    "Read build cache data and write Bazel CAS data" permissions.
+# 3. Select "Access keys" on the left and then generate a new access key.
+# 4. In a .bazelrc file in your user’s home directory, add the following:
+#    common:develocity --remote_cache_header=Authorization=<<access key generated>>
+#    common:develocity --bes_header=Authorization=<<access key generated>>
```
Comment (dragos) [inline common-gradle-enterprise-bazel-configuration/.bazelrc:21]: Make sure to watch this ticket for when we fix it: https://github.com/gradle/ge/issues/29905

Comment (runningcode) [reply common-gradle-enterprise-bazel-configuration/.bazelrc:21]: On second thought, let's remove this entire section. We don't describe configuring the access keys for either Gradle or Maven and it also isnt part of the Gradle or Maven samples.

Comment (cdsap) [reply common-gradle-enterprise-bazel-configuration/.bazelrc]: removed

**Review by runningcode — APPROVED** [review]
thanks

**Pull request merged**